packages/pangea-node-sdk/src/services/base.ts

```diff
@@ -59,7 +59,7 @@ class BaseService {
     return await this.request.downloadFile(url);
   }
 
-  async pollResult(request_id: string): Promise<PangeaResponse<any>> {
+  async pollResult<R>(request_id: string): Promise<PangeaResponse<R>> {
```

Comment:
Wait, isn't this a breaking change?

Reply:
Now defaulting `R = any` to preserve backwards compat.